### DISCUSSION TITLE
Need help with my PR

### DISCUSSION THREAD

**Discussion by AdarshAcharya5** [TheAlgorithms/Python#9028] [Q&A] [answered]
Hello. I'm new to open source and have made my first contribution to this repo. Can anyone tell me why my PR #9027 is failing the tests?. At first I thought it was because I directly pushed the commits to the master branch, but the build (pull_request) test is still failing after creating a separate branch. Thanks!

**Comment by tianyizheng02** [comment, marked as answer]
If you click on "Details" for the build, you can read through the build logs and find the following:
```
=================================== FAILURES ===================================
___ [doctest] scaled_exponential_linear_unit.scaled_exponential_linear_unit ____
021 
022     Applies the Scaled Exponential Linear Unit function to each element of the vector.
023     Parameters : vector : np.ndarray
024                  alpha : float (default = 1.6732)
025                  _lambda : float (default = 1.0507)
026     Returns : np.ndarray
027     Formula : f(x) = _lambda * x if x > 0
028                      _lambda * alpha * (e**x - 1) if x <= 0
029     Examples :
030     >>> scaled_exponential_linear_unit(np.array([1.3, 3.7, 2.4]))
Expected:
    Output : np.array([1.36591, 3.88759, 2.52168])
Got:
    array([1.36591, 3.88759, 2.52168])

/home/runner/work/Python/Python/neural_network/activation_functions/scaled_exponential_linear_unit.py:30: DocTestFailure
```
As you can see, it's a minor formatting issue in your doctest. The doctest returns the numpy array as `array()` while you've given the expected output as `np.array()`. The test should pass once you fix this.

In the future, please keep discussions about PRs within the PRs themselves.
